controllers/auth.js

```diff
@@ -115,7 +118,7 @@ auth.register = (req, res, next) => {
       .then(([userID]) => {
         return res.send(201, {
           id: userID,
-          message: 'registered user'
+          message: 'Registered user successfully'
```

Comment:
Could this be `User successfully registered` to be consistent with the messages on the front end? Just because that's the sentence structure I used.